plugins/system/shortcut/shortcut.php

```diff
@@ -31,7 +31,7 @@ public function onBeforeCompileHead()
 
             if (!$wa->assetExists('script', 'shortcut'))
             {
-                $wa->registerScript('shortcut', $this->_basePath . '/js/shortcut.js', [], ['defer' => true]);
+                $wa->registerScript('shortcut', $this->_basePath . '/js/shortcut.es6.js', [], ['defer' => true]);
```

Comment:
This is wrong!
`shortcut.es6.js` is the SOURCE file, the final files are `shortcut.js`, shortcut.min.js` and `shortcut-es5.js`, shortcut-es5.min.js`. 

Also, you need to study how the Web Assets work and fix the JS source, you merged some of my code with parts of your code and that makes no sense.

Reply:
Thank you, I am working on it.